server/src/projects/projects.service.ts

```diff
@@ -126,10 +148,19 @@ export class ProjectsService {
       const projectWithContacts = {
         ...project,
         'project-applicants': projectApplicantsWithContacts,
+        'team-members': projectTeamMembers.map(member => ({
+          dcp_dcpprojectteamid: member.dcp_dcpprojectteamid,
+          name: member.dcp_name,
```

Comment:
How do we feel about renaming like this? Just because I know we like to keep field names as close to the names in CRM as possible -- maybe we should keep it as `dcp_name`?

Reply:
Really good point and goot catch here, taylor. I think dcp_name make sense here

Reply:
Totally. Renaming dcp_dcpprojectteam to team-members is actually okay as far as how we handle other entity relationships. Agree with the renaming. I'm going to get this in then refactor afterward. Thanks for pointing that out.